pyesbulk/__init__.py

```diff
@@ -183,25 +281,74 @@ def actions_tracking_closure(cl_actions):
                     # ... otherwise consider it successful.
                     successes += 1
             elif status == 400:
-                doc = {
+                try:
+                    exc_payload = resp["exception"]
+                except KeyError:
+                    pass
```

Comment:
Should we get some meaningful exc_payload here instead of just ignoring the KeyError? Otherwise we can't distinguish resp["exception"] == None from lack of resp["exception"] ... which I can imagine being a useful distinction in weird cases.

Reply:
I'll add the following comment which may help:

```
                    # We have an exception object in the response object
                    # which is not always JSON serializable, so we use
                    # `repr` to turn that exception into a serializable
                    # string while maintaining as much information about
                    # the exception as possible.
```

Reply:
My concern was whether the value of exc_payload in the `else:` below would give us a useful clue as to whether `resp["exception"]` == None, or was missing, because the `pass` seems to silently obscure that distinction.

Reply:
So here are the three cases we are trying to handle:

 1. the key `exception` is not in the `resp` dictionary -- do nothing, as we have no object to ensure it is JSON serializable
 2. the key `exception` is in the `resp` dictionary -- emit a `repr` of whatever value is contained under that dictionaries key
 3. the key `exception` is in the `resp` dictionary but is `None` -- using `repr` should emit the string `'None'`

Does that make it clear regarding `pass` behavior?  Or am I still missing what you are trying to communicate?